themes/default/layouts/shortcodes/cloud-intro.html

```diff
@@ -1,16 +1,26 @@
 <div>
     <p>
         Pulumi's infrastructure-as-code SDK helps you create, deploy, and manage {{ .Get 0 }} containers,
-        serverless functions, and infrastructure using real programming languages. The Pulumi {{ .Get 0 }} provider packages
+        serverless functions, and infrastructure using <a href="{{ relref . "/docs/intro/languages" }}">imperative  programming languages</a>. The Pulumi {{ .Get 0 }} provider packages
         and <a href="{{ relref . "/docs/reference/cli" }}">CLI</a>
         help you accomplish all these within minutes.
     </p>
     <p>
-        This guide will have you quickly provisioning infrastructure on {{ .Get 0 }} with Pulumi in your
-        favorite language.
-    </p>
-    <p>
-        To get started we will walk through installing Pulumi, installing your preferred language runtime (if needed),
-        and configuring Pulumi with {{ .Get 0 }}. Click the button below to get started.
+        For a quick example of how Pulumi deploys infrastructure on {{ .Get 0 }}, this tutorial takes you through the following steps to easily deploy a static website:
     </p>
+    <ol>
+        <li>Setting up and configuring Pulumi to access your {{ .Get 0 }} account.</li>
+        <li>Creating a new Pulumi project.</li>
+        {{ if eq (.Get 0) "AWS" }}
+            <li>Provisioning a new AWS S3 bucket.</li>
+            <li>Adding an <code>index.html</code> file to your bucket and serving it as a static website.</li>
+        {{ else if eq (.Get 0) "Microsoft Azure" }}
+            <li>Provisioning a new Blob storage container.</li>
+            <li>Adding an <code>index.html</code> file to your container and serving it as a static website.</li>
+        {{ else }}
```

Comment:
This is okay, but it's not quite an "else" situation, since it only really works for Google Cloud.

```suggestion
        {{ else if eq (.Get 0) "Google Cloud" }}
```

Reply:
Added this in, but used the local `$cloud` variable per your earlier suggestion.